pkg/app/api/service/pipedservice/service.proto

```diff
@@ -362,9 +362,16 @@ message GetLatestEventResponse {
     pipe.model.Event event = 1 [(validate.rules).message.required = true];
 }
 
+enum ListEventsOrder {
+    NONE = 0;
+    FROM_OLDEST = 1;
+    FROM_NEWEST = 2;
+}
```

Comment:
How about `ListOrder` enum that contains `NONE`, `DESC`, `ASC`?

Reply:
That sounds good, that's more versatile. I initially thought about making it but I wasn't sure which should be general, CreatedAt or UpdatedAt, that's why I made a special enum for now. but what do you think?

Reply:
I think it is fine to make `ListOrder`. It can be used by other RPCs too.
Let's do it.

Reply:
Fine, I'll do so!